content/docker-hub/access-tokens.md

```diff
@@ -65,10 +65,16 @@ a password.
 You can rename, activate, deactivate, or delete a token as needed.
 
 1. Access your tokens under **Account Settings > Security**.
-   This page shows an overview of all your tokens. You can also view the number
+   This page shows an overview of all your tokens, and lists if the token was generated manually or if it was [auto-generated](#auto-generated-tokens). You can also view the number
    of tokens that are activated and deactivated in the toolbar.
```

Comment:
Not part of the diff, but not sure what `You can also view the number of tokens that are activated and deactivated in the toolbar` means. I don't see a toolbar in the Hub UI - can this be removed?

Reply:
Also cc @rossipedia maybe you know what this line is referring to in the UI.

Reply:
Merging this as-is for now so there's more info available about the auto-generated tokens, will address this section potentially in a future PR if needed.